app/controllers/coronavirus/pages_controller.rb

```diff
@@ -74,7 +53,7 @@ def redirect_to_index_if_slug_unknown
     end
 
     def publish_page
-      Services.publishing_api.publish(page.content_id, update_type)
+      Services.publishing_api.publish(page.content_id, "minor")
       change_state("published")
```

Comment:
Should this be inlined as per the GithubContentController? 

Also, a problem for another day, but I wonder if maybe we should think about moving this code into a service. it seems one of those annoying things where it is a small amount of code to keep in sync which is a bit dubious, but not really enough to justify doing anything with.

Also while you're clearing this file up. I think nothing makes use of: https://github.com/alphagov/collections-publisher/blob/779caa2a05c7fd3e632fda35bf10789af58afaa5/app/controllers/coronavirus/pages_controller.rb#L63-L72 anymore.

Reply:
I'm going to say all of this is out of scope for this change. That's why I specifically said I wasn't going to clean up the PagesController any further as part of this change as it's a very slippery slope down the rabbit hole.

Reply:
I can understand your concern. I'd have thought only the service thing is a pain to do here (which I wasn't expecting as part of this PR - perhaps when I said another day I should have been more explicit as some undefined time in the future). I'd have thought it'd be very quick to remove change_state and the `with_longer_timeout` method.

Reply:
There are quite a few little things that should be fixed in `PagesController`, that's why I tried to be disciplined in the changes in this PR and limit them only to things that are directly related to the Github changes page.

I plan to fix the niggles with `PagesController` in a separate PR.